config/schema.sql

```diff
@@ -7,27 +7,11 @@
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
-SET idle_in_transaction_session_timeout = 0;
```

Comment:
This line is a Postgres 9.6 command that has no meaning for Postgres 9.5; it snuck in because of using pg_dump 9.6.